support/core.go

```diff
@@ -98,7 +101,10 @@ func storeContainerLog(t Test, namespace *corev1.Namespace, podName, containerNa
 
 	options := corev1.PodLogOptions{Container: containerName}
 	stream, err := t.Client().Core().CoreV1().Pods(namespace.Name).GetLogs(podName, &options).Stream(t.Ctx())
-	t.Expect(err).NotTo(gomega.HaveOccurred())
+	if err != nil {
+		t.T().Logf("Pod Container %s/%s/%s not found, logs cannot be stored", namespace.Name, podName, containerName)
```

Comment:
This is more likely to fail when the Pod is still in pending phase, rather than because the container cannot be found.  A more generic message may be better suited not to be misleading.

Reply:
Adjusted wording.
I suppose it doesn't have sense to print the error, considering that this is not a test failure.